engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -120,11 +123,15 @@ private void createFBOs() {
         }
     }
 
-    // TODO: Pairing FBOs for swapping functionality
     public void swapReadWriteBuffers() {
-        FBO fbo = get(READONLY_GBUFFER);
-        fboLookup.put(READONLY_GBUFFER, get(WRITEONLY_GBUFFER));
-        fboLookup.put(WRITEONLY_GBUFFER, fbo);
-        notifySubscribers();
+        buffersSwapped = !buffersSwapped;
```

Comment:
I'll try and discuss this via slack rather than here. There might be misunderstandings that it's better to clear in an interactive session.